spec/models/user_spec.rb

```diff
@@ -143,32 +145,6 @@
   end
 
   describe "#send_welcome_email (after_save callback)" do
-    context "when the user does NOT have an email address" do
```

Comment:
I don't think this applies anymore as users will always have to have an email address @jpallen ?

Reply:
👍